profiles/console-ui/main.tf

```diff
@@ -8,9 +8,9 @@ module "console-ui" {
   filmdrop_ui_release    = var.console_ui_inputs.filmdrop_ui_release
   console_ui_bucket_name = var.console_ui_inputs.deploy_cloudfront ? module.cloudfront_s3_website[0].content_bucket_name : module.content_website[0].content_bucket
 
-  filmdrop_ui_config    = filebase64(var.console_ui_inputs.filmdrop_ui_config_file)
+  filmdrop_ui_config    = var.console_ui_inputs.filmdrop_ui_config_file != "" ? filebase64(var.console_ui_inputs.filmdrop_ui_config_file) : "bm9uZQo=" # None
   filmdrop_ui_logo_file = var.console_ui_inputs.filmdrop_ui_logo_file
-  filmdrop_ui_logo      = filebase64(var.console_ui_inputs.filmdrop_ui_logo_file)
+  filmdrop_ui_logo      = var.console_ui_inputs.filmdrop_ui_logo != "" ? var.console_ui_inputs.filmdrop_ui_logo : filebase64(var.console_ui_inputs.filmdrop_ui_logo_file)
```

Comment:
Not exactly sure if this would work without an existing logo file...

Reply:
I think the logic is correct the way I have it -- if `filmdrop_ui_logo` is not empty (the base64-encoded bytes for the logo file), then use it; otherwise, get the base64 of the contents of the filmdrop_ui_logo_file.

I don't think this will actually work in a meaningful way, but the current code won't run correctly with `terraform validate` because it's defaulting to `.` being the location of the file, which doesn't exist.

Reply:
The other option is to set this for all of them:

```
    filmdrop_ui_config_file = "./profiles/console-ui/default-config/config.dev.json"
    filmdrop_ui_logo_file   = "./profiles/console-ui/default-config/logo.png"
```